From 5f1baf6dd4a0e81e26f887e3c53743b3c7306c6b Mon Sep 17 00:00:00 2001
From: miyuki <miyuki@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Sun, 31 May 2015 08:12:55 +0000
Subject: [PATCH 09/28] Fix C++11 compatibility issue

	* config/cris/cris.h (CRIS_ARCH_CPP_DEFAULT): Fix C++11 compatibility
	issue (add space between string literal and macro).
	* config/i386/mingw32.h (REAL_LIBGCC_SPEC): Likewise.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@223898 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/config/cris/cris.h    | 2 +-
 gcc/config/i386/mingw32.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gcc/config/cris/cris.h b/gcc/config/cris/cris.h
index cb7baf7..e4652d6 100644
--- a/gcc/config/cris/cris.h
+++ b/gcc/config/cris/cris.h
@@ -134,7 +134,7 @@ extern int cris_cpu_version;
    %{!metrax*:\
     %{!mcpu=*:\
      %{!mtune=*:-D__tune_v" CRIS_DEFAULT_TUNE "}\
-     -D__arch_v"CRIS_DEFAULT_TUNE\
+     -D__arch_v" CRIS_DEFAULT_TUNE \
    " -D__CRIS_arch_version=" CRIS_DEFAULT_TUNE "}}}"
 #endif
 
diff --git a/gcc/config/i386/mingw32.h b/gcc/config/i386/mingw32.h
index 0950304..47e12e7 100644
--- a/gcc/config/i386/mingw32.h
+++ b/gcc/config/i386/mingw32.h
@@ -139,7 +139,7 @@ along with GCC; see the file COPYING3.  If not see
 #undef REAL_LIBGCC_SPEC
 #define REAL_LIBGCC_SPEC \
   "%{mthreads:-lmingwthrd} -lmingw32 \
-   "SHARED_LIBGCC_SPEC" \
+   " SHARED_LIBGCC_SPEC " \
    -lmoldname -lmingwex %{!mcrtdll=*:-lmsvcrt} %{mcrtdll=*:-l%*}"
 
 #undef STARTFILE_SPEC
-- 
2.5.1

